packages/documentation/src/stories/accessibility-practices/foundational-structure-and-semantics/reference-crossing-the-shadowdom/aria-labelledby/examples/arialabelledbyElementsProperty.sample.html

```diff
@@ -0,0 +1,3 @@
+<span id="labelId"></span>
+
+<element id="elementId"></element>
```

Comment:
Add some text content and use either a native element or a valid custom-element name for the `<element>` tag.